capture_gui/colorpicker.py

```diff
@@ -0,0 +1,47 @@
+from capture_gui.vendor.Qt import QtCore, QtWidgets, QtGui
+
+
+class ColorPicker(QtWidgets.QPushButton):
+    """Custom color pick button to store and retrieve color values"""
+
+    def __init__(self):
+        QtWidgets.QPushButton.__init__(self)
+
+        self.clicked.connect(self.get_color_value)
+        self._color = None
+
+        self.color = [1, 1, 1]
+
+    # region properties
+    @property
+    def color(self):
+        return self._color
+
+    @color.setter
+    def color(self, values):
+        """Set the color value and update the stylesheet
+        
+        :param values: the color values; red, green, blue
+        :type values: list
+        
+        :return: None
+        """
+        self._color = values
+
+        values = [int(x*255) for x in values]
+        self.setStyleSheet("background: rgb({},{},{})".format(*values))
+
+    # endregion properties
+
+    def get_color_value(self):
```

Comment:
This method function and docstring sounds somewhat confusing, because it sounds like it is _just getting the color_. Reading only that I'd expect it to return the currently set color, but instead it's a method to show a dialog to set a new color.

Maybe this is better as `on_set_color_value` or `set_color_dialog`?